### ISSUE TITLE
Sharpness in Table 1 from the paper

### ISSUE THREAD

**Issue by Billy1900** [wenwei202/smoothout#2]
I met a little problem. I am confused about that what does it mean if the sharpness is 0 (in your Table 1)? Does it mean absolute flat?

**Comment by wenwei202** [comment]
It's floating-point precision issue, which means the sharpness is pretty small.

**Comment by Billy1900** [comment]
Yeah, but 0? I was a little doubtful. According to your paper, your sharpness metric is based on 'ON LARGE-BATCH TRAINING FOR DEEP LEARNING: GENERALIZATION GAP AND SHARP MINIMA' as follows:
![1MBZEA 8RW@CQ381R}NJL15](https://user-images.githubusercontent.com/32431813/99870386-886d6600-2c0d-11eb-8b29-9ba1a66d33a9.png)
if the result is 0, it means that the local region is pretty flat even like a plane?

**Comment by wenwei202** [comment]
Correct. It is like a plane within the small bounding box. It's not uncommon in neural nets.
